RELEASENOTES.docu

```diff
@@ -1761,4 +1761,12 @@ extern typedef struct { } my_type_t;</pre> </add-note></build-id>
       <tt>typeof</tt> within a method signature would lead to incorrect
       compile-time errors when the method is overridden.</add-note>
   </build-id>
+  <build-id _6="next" _7="next"><add-note> DMLC will now report errors
+      for some uses of function types that are illegal in C. These
+      would previously yield broken C code.</add-note></build-id>
```

Comment:
Why not mention the compatibility flag?

Reply:
The compatibility feature is mentioned in the releasenote just below this comment.

Reply:
Uhhh
I have no excuse.